src/lib/components/SideNav/_styles.scss

```diff
@@ -39,6 +39,12 @@ web-side-nav {
   z-index: 300;
 }
 
+// Prevent FOUC by hiding children until after firstUpdated(), as children are
+// reparented within .web-side-nav__container before use
+web-side-nav.unresolved * {
```

Comment:
Could we add `:not(:defined)` here as well?

Reply:
No, because there's a change after a frame (due to reparenting) and so the flash still occurs. I think maybe Chrome is showing me a frame before lit renders.

Reply:
I've added a comment to this effect.